### ISSUE TITLE
CN64System::RunRSP() ; Unkown memory action

### ISSUE THREAD

**Issue by SandwichWithHam** [Rosalie241/BetterMajorasMaskInstaller#12]
Similar to a previous issue posted by @carlcorder , I installed Project64 with your tool. When trying to launch MM, I get the following error; 

CN64System::RunRSP() 
Unkown memory action

![Screenshot 2021-03-30 140826](https://user-images.githubusercontent.com/81645043/113035647-6eb2d100-9161-11eb-9c21-15e7777849d4.png)

I followed all the steps taken in the previous issue, like double-checking the plugins and changing the audio driver.

I am not familiar with Project64, so I am not sure where to start.

**Comment by carlcorder** [comment]
Hi @SandwichWithHam, were you able to verify the ROM works without the texture pack being applied?

**Comment by SandwichWithHam** [comment]
Yes. 
When using Project64 v2.3.2, I was able to load the MM ROM, as well as a patched ROM I was trying to use.

If this is of any help, I decided to use this installer to get the HD textures to work. When using a regular installation with Gliden64 graphics drivers, I wasn't able to load the HD textures. 
I'm would still like to use this installer, but perhaps there might be a detail in that which would help.
